frontend/src/robot/capabilities/Wifi.tsx

```diff
@@ -0,0 +1,213 @@
+                {downspeed && <Typography variant="caption" sx={{mr: 1}}>⇩ {downspeed} MBit/s</Typography>}
+                {frequency && <Typography variant="caption" sx={{mr: 1}}>{frequencyStr}</Typography>}
+                {signal && <Typography variant="caption">{signal} dBm</Typography>}
+
+                <Box/>
+                {signal && <WifiProgress value={100 + signal} variant="determinate"/>}
+                {ips && <Typography variant="caption">IPs: {ips}</Typography>}
+
+                <Accordion sx={{mt: 1}}>
+                    <AccordionSummary
+                        expandIcon={<ExpandMoreIcon/>}
+                    >
+                        <Typography>Change Wifi connection</Typography>
+                    </AccordionSummary>
+                    <AccordionDetails>
+                        <TextField label="SSID/Wifi name" variant="standard" fullWidth
+                            value={newSSID} sx={{mb: 1}}
+                            onChange={(e) => {
+                                setNewSSID(e.target.value);
+                            }}/>
+                        <TextField label="PSK/Password" variant="standard" type="password" fullWidth
+                            value={newPSK} sx={{mb: 1}}
+                            onChange={(e) => {
+                                setNewPSK(e.target.value);
+                            }}/>
+                        <LoadingButton loading={wifiConfigurationUpdating} variant="outlined" color="success"
+                            disabled={!newSSID || !newPSK} onClick={() => {
+                                setConfirmationDialogOpen(true);
+                            }}>Apply</LoadingButton>
+                    </AccordionDetails>
+                </Accordion>
+
+                <ConfirmationDialog title="Apply new Wifi configuration?" text="" open={confirmationDialogOpen}
+                    onClose={() => {
+                        setConfirmationDialogOpen(false);
+                    }} onAccept={() => {
+                        setFinalDialogOpen(true);
+                        updateWifiConfiguration({
+                            ssid: newSSID,
+                            credentials: {
+                                type: "wpa2_psk",
+                                typeSpecificSettings: {
+                                    password: newPSK
+                                }
+                            }
+                        });
+                    }}>
+                    <DialogContentText>
+                        Are you sure you want to apply the new wifi settings?
+                        <br/>
+                        <strong>Hint:</strong> You can always revert back to the integrated Wifi Hotspot
+                        <ul>
+                            <li>on Dreame: By pressing both the home and spot cleaning button for a few seconds</li>
```

Comment:
I'd say that we can omit these because that might get a bit too excessive relatively fast with new models

Reply:
Sounds good